roles/redis/templates/redis.secret.yaml.j2

```diff
@@ -17,3 +17,4 @@ stringData:
   port: '6379'
   host: {{ ansible_operator_meta.name }}-redis-svc
   type: 'managed'
+  redis_mode: 'standalone'
```

Comment:
This would require to handle a new variable, but we already know if it is a cluster with the current instrumentation based on EDA_MQ_REDIS_HA_CLUSTER_HOSTS (cluster_endpoint) What do you think?

Reply:
Hmmm. I can look into that implementation of `EDA_MQ_REDIS_HA_CLUSTER_HOSTS` to check. I did see that when redis is installed in `cluster` mode, that this secret does have a `redis_mode` value of `cluster`. 

However our default is standalone. I figured initially that setting it to `standalone` here made it more consistent. However, I can look into using this variable instead to check if Redis is in cluster/standalone mode if that is recommended here.